README.md

```diff
@@ -534,6 +534,34 @@ export default (config, env) => {
 
 When you build your Preact application this will run `extractCss` on your prerendered pages and add critical styles for each page.
 
+## CSS Prop
```

Comment:
Would you mind adding this as part of TOC at the top as well? Also, I think this would be sweet to be included in the generated docs at goober.js.org. So that means, this section needs to be copy/pasted into `/docs` folder in the hierarchy.

Reply:
Updated both changes